src/popup/components/AddressTruncated.vue

```diff
@@ -32,15 +37,18 @@ import { PROTOCOL_AETERNITY } from '@/constants';
 import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
 
 import ExternalLinkIcon from '@/icons/external-link.svg?vue-component';
+import ProtocolIcon from './ProtocolIcon.vue';
 import LinkButton from './LinkButton.vue';
 
 export default defineComponent({
   components: {
+    ProtocolIcon,
     LinkButton,
     ExternalLinkIcon,
   },
   props: {
     address: { type: String, required: true },
+    showProtocolIcon: Boolean,
```

Comment:
Please move this line below the `protocol`. The reason is that it's a good pattern to group Boolean props together. Also it makes more sense in terms of alphabetical order.